supervision/tracker/byte_tracker/byte_tracker.py

```diff
@@ -0,0 +1,483 @@
+from typing import List
+
+import numpy as np
+from onemetric.cv.utils.iou import box_iou_batch
```

Comment:
We don't need to use `onemetric`. We have `box_iou_batch` in `supervision`. You can find it in `supervision.detection.util`.